### DISCUSSION TITLE
How to rename columns while querying in supabase.

### DISCUSSION THREAD

**Discussion by hitesh-c** [supabase/supabase#29451] [Questions] [answered]
I need Supabase-js code equivalent to this SQL query 

```SQL
SELECT 
  column_name1 AS custom_name1,
  column_name2 AS custom_name2,
  column_name3 AS custom_name3
FROM 
  table_name;
```

**Comment by GaryAustin1** [comment, marked as answer]
Aliases are supported.   .select('columnCustomA:column_a')

**Reply by hitesh-c** [reply]
Ok custom name should be given at first? Thanks

**Reply by pentolbakso** [reply]
Is there anywhere in the docs where I can learn more about it? supabase or postgres

**Reply by GaryAustin1** [reply]
https://docs.postgrest.org/en/v12/references/api/tables_views.html#renaming-columns

**Comment by hitesh-c** [comment]
Thanks
